pipeline/backend/docker/docker.go

```diff
@@ -318,10 +318,10 @@ func (e *docker) DestroyWorkflow(_ context.Context, conf *backend.Config, taskUU
 		for _, step := range stage.Steps {
 			containerName := toContainerName(step)
 			if err := e.client.ContainerKill(noContext, containerName, "9"); err != nil && !isErrContainerNotFoundOrNotRunning(err) {
-				log.Error().Err(err).Msgf("could not kill container '%s'", stage.Name)
+				log.Error().Err(err).Msgf("could not kill container '%s'", step.Name)
```

Comment:
nice catch but same as first coment

Reply:
that nit could stay in ... but I also put it into https://github.com/woodpecker-ci/woodpecker/pull/2995 as it lowers the pull diff here